mteb/abstasks/AbsTaskSpeedTask.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import logging
+import platform
+import time
+from pathlib import Path
+
+import GPUtil
+import numpy as np
+import psutil
+
+from mteb.encoder_interface import Encoder, EncoderWithQueryCorpusEncode
+from mteb.MTEBResults import ScoresDict
+
+from .AbsTask import AbsTask
+
+logger = logging.getLogger(__name__)
+
+
+TOKENS_IN_UGLY_DUCKLING = 3591
```

Comment:
Is this `WORDS_IN..`, as it's used for `"avg_words_per_sec": TOKENS_IN_UGLY_DUCKLING / time_mean,`?
If it's tokens I think we should specify the tokenizer but then it's probably dependent on the model that is used, no?

Reply:
Yeah good call. Just counted (in Google Docs) 3633 words. I took this directly from the Scandinavian Embedding Benchmark - maybe @KennethEnevoldsen or @x-tabdeveloping might know how this number was obtained? Otherwise I'm fine with changing to 3633 words.

Reply:
I think it is word either using the spacy tokenizer for Danish or whitespace split. Using `3633` is probably better.

Reply:
Changed to `WORDS...` and value to 3633.